bindings/java/src/org/sleuthkit/datamodel/BlackboardArtifact.java

```diff
@@ -316,7 +561,7 @@ public int hashCode() {
 	 */
 	@Override
 	public String toString() {
-		return "BlackboardArtifact{" + "artifactID=" + artifactId + ", objID=" + objId + ", artifactTypeID=" + artifactTypeId + ", artifactTypeName=" + artifactTypeName + ", displayName=" + displayName + ", Case=" + sleuthkitCase + '}'; //NON-NLS
+		return "BlackboardArtifact{" + "artifactID=" + artifactId + ", objID=" + getObjectID() + ", artifactObjID=" + artifactObjId + ", artifactTypeID=" + artifactTypeId + ", artifactTypeName=" + artifactTypeName + ", displayName=" + displayName + ", Case=" + getSleuthkitCase() + '}'; //NON-NLS
```

Comment:
This isn't applicable any more if we BlackboardArtifact isn't extending AbstractContent. Right?